web/beacon-app/src/constants/app.ts

```diff
@@ -28,3 +28,26 @@ export const QDK_API_ROUTE = {
   REGISTER: '/register',
   REFRESH_TOKEN: '/refresh',
 };
+
+// mime type constants
+export enum MIME_TYPE {
```

Comment:
Will the user/format MIME types listed [here](https://github.com/rotationalio/ensign/blob/develop/pkg/ensign/mimetype/v1beta1/mimetype.go#L90-L118) need to be added?